sparsity/sparse_frame.py

```diff
@@ -108,9 +108,10 @@ def todense(self, pandas=True):
         else:
             dense = np.empty(shape=(0, len(self.columns)))
 
-        if self.shape[0] == 1 or self.shape[1] == 1:
-            dense = dense.reshape(-1)
-        if pandas == True:
+        if pandas:
+            if self.shape[0] == 1 or self.shape[1] == 1:
+                dense = dense.reshape(-1)
+
```

Comment:
This change breaks backwards compatibility. Why would you reshape only if `pandas==True`.